src/arctic3d/modules/interface_matrix.py

```diff
@@ -239,13 +241,15 @@ def interface_matrix(interface_dict, pdb_path, int_cov_cutoff=0.7):
     if not os.path.exists(pdb_path):
         raise Exception(f"pdb_path {pdb_path} does not exist")
     mdu = mda.Universe(pdb_path)
-    pdb_resids = mdu.select_atoms("name CA").resids
+    pdb_resids = mdu.select_atoms(
+        f'protein and name CA and not icode in {" ".join(LETTERS)}'
+    ).resids
     retained_interfaces = filter_interfaces(
         interface_dict, pdb_resids, int_cov_cutoff
     )
-    print(f"retained_interfaces: {retained_interfaces}")
     ret_keys = list(retained_interfaces.keys())
-    log.debug(f"Retained interfaces: {ret_keys}")
+    if len(ret_keys) < 10:
```

Comment:
Could you add a comment as to why 10 here?

Reply:
good point, was to avoid printing too many interfaces